packages/roosterjs-editor-core/lib/corePlugins/createCorePlugins.ts

```diff
@@ -32,7 +31,7 @@ export default function createCorePlugins(
         typeInContainer: map.typeInContainer || new TypeInContainerPlugin(),
         edit: map.edit || new EditPlugin(),
         _placeholder: null,
-        typeAfterLink: map.typeAfterLink || new TypeAfterLinkPlugin(),
+        typeAfterLink: null,
```

Comment:
You can set the value to null, but also need to mark this property as deprecated in CorePlugins.ts

Reply:
Please add comment why we set to null.